### ISSUE TITLE
Needs more systems

### ISSUE THREAD

**Issue by new-penguin** [lipebello/Retroarch-XMB-Retrorama-Theme#2]
Maybe I'm abusing the issue system here but I love the theme, it's what I've been using since I saw your upload on libretro forums.  But covering a few more systems would be great, like Gamecube, Wii, Wii U, PS2, Colecovision, etc. But it's hard to complain about free media. Thanks.

**Comment by jtyall** [comment]
I agree!

**Comment by LeoLeal** [comment]
+1 one the other systems.

Great work indeed!

**Comment by DiazExMachina** [comment]
I agree, it's missing some common systems like GameCube and PlayStation 2 (that I know of, at least)
